src/main/java/org/opensearch/security/tools/SecurityAdmin.java

```diff
@@ -1379,15 +1378,177 @@ private static boolean validateConfigFile(String file, CType cType, int version)
             System.out.println(file+" OK" );
             return true;
         } catch (Exception e) {
-            System.out.println("ERR: Seems "+file+" is not in SG "+version+" format: "+e);
+            System.out.println("ERR: Seems "+file+" is not in "+version+" format: "+e);
             return false;
         }
     }
-    
-    private static String[] getTypes(boolean legacy) {
-        if(legacy) {
-            return new String[]{"config","roles","rolesmapping","internalusers","actiongroups","nodesdn", "audit"};
-        }
-        return CType.lcStringValues().toArray(new String[0]);
-    }
+
+	private static String[] getTypes(boolean legacy) {
+		if (legacy) {
+			return new String[]{"config", "roles", "rolesmapping", "internalusers", "actiongroups", "nodesdn", "audit"};
+		}
+		return CType.lcStringValues().toArray(new String[0]);
+	}
+
+
+	private static RestHighLevelClient getRestHighLevelClient(SSLContext sslContext,
+															  boolean nhnv,
+															  String[] enabledProtocols,
+															  String[] enabledCiphers,
+															  String hostname,
+															  int port) {
+
+		final HostnameVerifier hnv = !nhnv ? new DefaultHostnameVerifier() : NoopHostnameVerifier.INSTANCE;
+
+		String[] supportedProtocols = enabledProtocols.length > 0 ? enabledProtocols : null;
+		String[] supportedCipherSuites = enabledCiphers.length > 0 ? enabledCiphers : null;
+
+		HttpHost httpHost = new HttpHost(hostname, port, "https");
+
+		RestClientBuilder restClientBuilder = RestClient.builder(httpHost)
+				.setHttpClientConfigCallback(
+						builder -> builder.setSSLStrategy(
+								new SSLIOSessionStrategy(
+										sslContext,
+										supportedProtocols,
+										supportedCipherSuites,
+										hnv
```

Comment:
The format looks a bit off. Is it formatted by IDE?

Reply:
Formatting was a bit difficult for us as different files seem to use different formattings. Do you have a standard formatting policy?

Reply:
I see. This format is good then. It is consistent with the format in this particular file.